modules/elementor-counter/module.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Elementor\Modules\ElementorCounter;
+
+use Elementor\Core\Isolation\Elementor_Counter_Adapter_Interface;
+use Elementor\Core\Isolation\Wordpress_Adapter;
+use Elementor\Core\Isolation\Wordpress_Adapter_Interface;
+use Elementor\Core\Base\Module as BaseModule;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+class Module extends BaseModule implements Elementor_Counter_Adapter_Interface {
```

Comment:
I am wondering if it makes sense to have this as a 'module', as it's not a feature. It could be split into a 'component' of the checklist?

Reply:
Tzvi said that we should change it to be more global than the checklist because by definition the editor counter is something that we should count regardless of it. We might want to later expand that counter to other areas